src/auth-service/models/Scope.js

```diff
@@ -0,0 +1,231 @@
+const mongoose = require("mongoose").set("debug", true);
+const { logObject } = require("@utils/log");
```

Comment:
Adding the scope model, should the network ID also be required here? @Codebmk / @123MwanjeMike / @wabinyai